### DISCUSSION TITLE
Think about the MySQL collation

### DISCUSSION THREAD

**Discussion by bschmalhofer** [RotherOSS/otobo#1019] [Q&A]
https://stackoverflow.com/questions/766809/whats-the-difference-between-utf8-general-ci-and-utf8-unicode-ci suggest that _utf8mb4_unicode_ci_ might be a better collation than _utf8_general_ci_.

**Comment by bschmalhofer** [comment]
There is also the shiny new _utf8mb4_0900_ai_ci_ but probably not yet on MariaDB. Anyways, let's move this to the next release.

**Comment by wollmers** [comment]
`utf8mb4_unicode_ci`  is the correct one.

On newer MySql (>= 8.0):

https://mysqlserverteam.com/mysql-8-0-collations-migrating-from-older-collations/#:~:text=From%20MySQL%208.0%2C%20utf8mb4%20is,of%20the%20complete%20Unicode%209.0.

```
From MySQL 8.0, utf8mb4 is the default character set, and the default collation for utf8mb4 is utf8mb4_0900_ai_ci. 
MySQL 8.0 is also coming with a whole new set of Unicode collations for the utf8mb4 character set.
```

MariaDB (<= 10.2.2) has `utf8mb4_unicode_ci`, i.e. on the level of Unicode Version 4.
